gettsim/tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+import yaml
+
+from gettsim.config import ROOT_DIR
+
+
+@pytest.fixture(scope="session")
+def tax_policy_data():
+    return yaml.safe_load(open(ROOT_DIR / "data" / "param.yaml", "rb"))
```

Comment:
No need for open(). Just use read_text() as a path method.

Reply:
Did it!